include/deal.II/fe/fe_dgp_monomial.h

```diff
@@ -357,18 +357,19 @@ class FE_DGPMonomial : public FE_Poly<PolynomialsP<dim>, dim>
   hp_constraints_are_implemented() const override;
 
   /**
-   * Return whether this element dominates the one given as argument when they
-   * meet at a common face, whether it is the other way around, whether
-   * neither dominates, or if either could dominate.
-   *
-   * For a definition of domination, see FiniteElementDomination::Domination
-   * and in particular the
-   * @ref hp_paper "hp paper".
+   * @copydoc FiniteElement::compare_for_face_domination()
    */
-  virtual FiniteElementDomination::Domination
+  DEAL_II_DEPRECATED virtual FiniteElementDomination::Domination
   compare_for_face_domination(
     const FiniteElement<dim> &fe_other) const override;
```

Comment:
hm, if you leave these functions here as is, then you can't do the `final` tag in the base class. So maybe ignore my comment above.